### DISCUSSION TITLE
Open Port on Music Assistant

### DISCUSSION THREAD

**Discussion by ItsCharged** [music-assistant/support#2158] [Q&A] [answered]
I really want to open a port on music assistant so that i can acess it from the outside however it shows no options for this in the config of the addon and if i try to open 192.168.XXX.XXX:8095 Nothing happens. Same thing goes for Homeassistant.local:8095 

Trying to open addon
![image](https://github.com/music-assistant/support/assets/90069597/c67c583f-69dd-437b-8c14-52c75312f881)

Looking in addon config
![image](https://github.com/music-assistant/support/assets/90069597/fff51a92-ed56-4d23-9836-c3909bab3aad)

Trying older /lib/mass
![image](https://github.com/music-assistant/support/assets/90069597/dd1d54f9-2836-4ca5-a476-2926e623ce94)

**Comment by OzGav** [comment, marked as answer]
We STRONGLY recommend you do not do that as there is no security. Enabling that port is done in the settings.

**Reply by ItsCharged** [reply]
Thanks! I really dont care, as the way to acess that is my private homeassistant loop wifi or tailscale acc. and the wifi is only linked to personal devices and protected with a strong 16 char. password
